src/main/kotlin/LinkedList.kt

```diff
@@ -10,18 +10,54 @@ class LinkedList<T> {
     
     fun count(): Int = size
        
-    fun insert(index: Int, obj: T) {
-        TODO("Not yet implemented")
+    operator fun get(index: Int): T {
+        var current = head
+        var currentIndex = 0
+        while (current != null && currentIndex < index) {
+            current = current.nextNode
+            currentIndex++
+        }
+        return current
     }
```

Comment:
Это получение ноды, а не элемента.
```suggestion
    private fun getNode(index: Int): Node<T> {
        var current = head
        var currentIndex = 0
        while (current != null && currentIndex < index) {
            current = current.nextNode
            currentIndex++
        }
        return current!!
    }
    operator fun get(index: Int): T {
        require(index in 0 until size-1)
        return getNode(index).value
    }
```